ctapipe_io_lst/tests/test_pointing.py

```diff
@@ -0,0 +1,38 @@
+import os
+from pathlib import Path
+from astropy.time import Time
+import astropy.units as u
+
+test_data = Path(os.getenv('LSTCHAIN_TEST_DATA', 'test_data'))
+test_drive_report = test_data / 'real/monitoring/DrivePositioning/drive_log_20200218.txt'
+
+
+def test_read_drive_report():
+    from ctapipe_io_lst.pointing import PointingSource
+
+    drive_report = PointingSource._read_drive_report(test_drive_report)
+
+    assert 'time' in drive_report.colnames
+    assert 'azimuth_avg' in drive_report.colnames
+    assert 'zenith_avg' in drive_report.colnames
+
+
+def test_interpolation():
+    from ctapipe_io_lst.pointing import PointingSource
+    from ctapipe_io_lst import LSTEventSource
+
+    subarray = LSTEventSource.create_subarray(geometry_version=4)
+    pointing_source = PointingSource(
+        subarray=subarray,
+        drive_report_path=test_drive_report,
+    )
+
+    time = Time('2020-02-18T21:40:21')
+    # El is really zenith distance
```

Comment:
omg :D ...

Reply:
Unfixed since half a year I might add

Reply:
:D .. let's close this .. it was a stupid comment by me.